extension/content.css

```diff
@@ -288,6 +288,30 @@ svg.octicon.octicon-star.dashboard-event-icon {
 	color: inherit !important;
 }
 
+/* Created repository style in dashboard */
```

Comment:
> Does xo lint css also?

No, but install an EditorConfig plugin and it will handled automagically: http://editorconfig.org